src/pages/Commands.tsx

```diff
@@ -85,6 +93,8 @@ export default () => {
 		});
 	}, []);
 
+	const parseDescription = (description: string) => description.replace(/<:shiny:[0-9]{18}>/gi, 'shiny');
```

Comment:
Please place a TODO in here, it should render images instead. For now you can do:
```suggestion
	const parseDescription = (description: string) => description.replace(/<:(\w{2,32}):[0-9]{18}>/gi, '$1');
```